docs/src/CodeMirror.css

```diff
@@ -0,0 +1,15 @@
+.cm-s-solarized.CodeMirror {
+  -moz-box-shadow: none;
+  -webkit-box-shadow: none;
+  box-shadow: none;
+}
+
+.highlight, .code-toggle.open {
+  background-color: #fdf6e3;
+}
+
+.cm-s-solarized .cm-comment { color: #93a1a1; }
+
+.CodeMirror {
+  height: auto;
```

Comment:
Looks like this line is the cause of the extra-tall editor height, through removing it causes other issues (scrolbars).